var/spack/repos/builtin/packages/py-neurokit2/package.py

```diff
@@ -19,9 +19,11 @@ class PyNeurokit2(PythonPackage):
     homepage = "https://github.com/neuropsychology/NeuroKit"
     pypi     = "neurokit2/neurokit2-0.1.2.tar.gz"
 
+    version('0.1.4.1', sha256='226bb04bb369d8bb87d99831f0a93cd8d0ed96fdc500f63de0b3550082876f6e')
     version('0.1.2', sha256='5ef40037c2d7078ecb713ab0b77b850267babf133856b59595de9613f29787bc')
 
     depends_on('py-setuptools@040.6.0:', type='build')
```

Comment:
```suggestion
    depends_on('py-setuptools@40.6.0:', type='build')
```
Just noticed this. I don't think it makes a difference, but might as well remove the typo

Reply:
Oh that's easy to overlook. Thanks!